src/pages/admin/student/edit/actions/spec/summaryStudentRequest.spect.ts

```diff
@@ -0,0 +1,101 @@
+import thunk from 'redux-thunk';
+import configureStore from 'redux-mock-store';
+
+import { adminActionEnums } from '../../../../../../common/actionEnums/admin';
+import { summaryStudentByIdRequestCompleted, summaryStudentByIdRequestStarted } from '../summaryStudentRequest';
+import { StudentSummary } from '../../../../../../model/studentSummary';
+import { studentApi } from '../../../../../../rest-api';
+
+const mockStore = configureStore([thunk]);
+
+describe('summaryStudentByIdRequestCompleted', () => {
+  it('should be an object', () => {
+    // Assert
+    expect(summaryStudentByIdRequestCompleted).not.to.be.an('object');
```

Comment:
Not to be an object? I think this should be a function because  `summaryStudentByIdRequestCompleted` is an ActionCreator that returns an action